arch/arc/core/CMakeLists.txt

```diff
@@ -30,3 +30,5 @@ zephyr_library_sources_ifdef(CONFIG_ARC_CONNECT arc_smp.c)
 
 add_subdirectory_ifdef(CONFIG_ARC_CORE_MPU mpu)
 add_subdirectory_ifdef(CONFIG_ARC_SECURE_FIRMWARE secureshield)
+
+zephyr_linker_sources(TEXT_START KEY 0x0vectors vector_table.ld)
```

Comment:
I'm a bit curious about the `0x0vectors` or `0x0<somename>` syntax is `<somename>` a value set before?